akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/ActorContext.scala

```diff
@@ -154,4 +156,9 @@ trait ActorContext[T] { this: akka.actor.typed.javadsl.ActorContext[T] ⇒
    */
   def spawnAdapter[U](f: U ⇒ T): ActorRef[U]
 
+  // FIXME make spawnAdapter an internal feature
+
+  // FIXME docs
+  def addMessageTransformer[U: ClassTag](f: U ⇒ T): ActorRef[U]
```

Comment:
I think we should change the name in a way that reflects that it might also replace the existing transformer. Maybe `handleOtherMessageType`?